database_reviews/fake-data.js

```diff
@@ -1,23 +1,27 @@
-const faker = require('faker')
-const fs = require('fs')
+const faker = require("faker");
+const fs = require("fs");
 
 faker.seed(100);
 
 let reviews = [];
 
 //generating 100 set of reviews
 for (let i = 0; i < 100; i++) {
-    let numberOfReviews = faker.random.number({min: 1, max: 20});
-    let numberOfWords = faker.random.number({min: 5, max: 15});
-    for (let j = 0; j < numberOfReviews; j++) {
-        reviews.push(`"${faker.random.words(numberOfWords)}"`, faker.random.number({min: 1, max: 5}), 
-        faker.random.number({min: 0, max: 1}), faker.name.firstName(), faker.date.recent(), i)
-    }
+  let numberOfReviews = faker.random.number({ min: 1, max: 20 });
+  let numberOfWords = faker.random.number({ min: 5, max: 15 });
```

Comment:
Is there a reason why you're determining the number of words each review should have outside the second for loop? By doing so, aren't you guaranteeing that for each review for a given product will have exactly number of words?

Reply:
Oh I did not notice this at all. Yes, you are absolutely right. I need to fix that. thank you!